ServiceBusDashboard/Code/SbConnectionStrings.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.IO;
+using System.Linq;
+using System.Web;
+using System.Web.Hosting;
+using Newtonsoft.Json;
+
+namespace ServiceBusDashboard.Code
+{
+    public class SbConnectionStrings
+    {
+        private static SbConnectionStrings _instance;
+        public SbConnectionString[] ConnectionStrings { get; private set; }
+
+        public static SbConnectionStrings Instance => _instance ?? (_instance = new SbConnectionStrings());
+
+        private SbConnectionStrings()
+        {
+
+        }
+
+        public void Load()
+        {
+            var rootPath = HostingEnvironment.MapPath("~/");
+            var connectionStringsContent = ConfigurationManager.AppSettings["SbConnectionStrings"];
```

Comment:
I think the name connectionStringsContent is a bit misleading? This value is actually the relative path to the connection strings file...

Reply:
I thought it would be good if the user can just paste the connection string inside the value, instead of a file path containing a single connection string.

Reply:
Changed it to connectionStrings.